### PR TITLE
Create Project Overview Notebook

### PR DESCRIPTION
**Pull request by BryanCutler**
This adds a notebook to show the basics and give an overview of Text Extensions for Pandas, and serve as a jumping off point to other notebooks.

### CODE REVIEW

**Comment by BryanCutler** [comment]
I still need to fill in some parts, but could use a review of this so far. cc @frreiss @MLnick @kmh4321

File: notebooks/Text_Extensions_for_Pandas_Overview.ipynb [hunk 1]
```diff
@@ -0,0 +1,1543 @@
+{
```
Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>convienent --&gt; convenient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>"begin, end values of the tokens" ==&gt; "begin and end offsets measured in tokens"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>Maybe include a reference to https://en.wikipedia.org/wiki/Tensor or to Numpy ndarrays here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>"will be incorporated with" =&gt; "supports many"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>It would be nice to have an introductory paragraph here that explains the file formats we support and where to find more information about those formats.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>You might want to fold this part into the <code>Span</code> section  above. Perhaps show a single example of something from the <code>spanner</code>  package, then refer to the API docs (https://text-extensions-for-pandas.readthedocs.io/en/latest/#module-text_extensions_for_pandas.spanner) for more information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (frreiss) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>I like having this section at the end. This is a perfect place to include "teasers" for each of our other notebooks, with links to the notebooks for the reader to find out more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (BryanCutler) [reply notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>I came up with an example, I think it flows pretty good but let me know what you think</p>

Comment (BryanCutler) [reply notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: I only talked about feather, I think that is all that work now right?

**Comment by BryanCutler** [comment]
@frreiss I filled in the rest of this notebook and addressed your feedback, please take another look - thanks! Also cc @kmh4321 , maybe you could take a quick look as someone newer to the project and see if everything is clear?

**Comment by kmh4321** [comment]
Sure, I'll also take a look.

File: notebooks/Text_Extensions_for_Pandas_Overview.ipynb (hunk 1 above)
Comment (BryanCutler) [reply notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: It does flow better to have this after the spans, so I moved it there.

Comment (frreiss) [reply notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: I think so, yes.

**Comment by frreiss** [comment]
The current version of this notebook looks good to me. Have you had a chance to review it, @kmh4321 ?

File: notebooks/Text_Extensions_for_Pandas_Overview.ipynb [hunk 2]
```diff
@@ -0,0 +1,1819 @@
...
```
Comment (kmh4321) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <ul><li>Might be good to add a hyperlink to Pandas DataFrames -&gt; https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (kmh4321) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <ul><li>Can we change the requirements to Python 3.6 now that we have support? </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (kmh4321) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>The first sentence is a bit confusing. maybe reword it to "Text Extensions for Pandas provides several Pandas extension arrays to the end user. Much of the functionality of this library is built on top of these extension arrays."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (kmh4321) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>Add a small blurb about the crude tokenizer in our case is just splitting the paragraph into individual words. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (kmh4321) [inline notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>This section seems a bit off theme to the overall NLP focus of this notebook. Maybe a line explaining a potential use of TensorArray might be helpful. Or any other rationale behind adding TensorArray support to TEP. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Review by kmh4321 — APPROVED** [review]
Overall looks great! My comments above are suggestions but not blockers as such.

**Comment by BryanCutler** [comment]
Thanks @frreiss and @kmh4321 , good suggestions! I'll update again tonight and merge when done, we can fine tune more later if needed.

File: notebooks/Text_Extensions_for_Pandas_Overview.ipynb (hunk 2 above)
Comment (BryanCutler) [reply notebooks/Text_Extensions_for_Pandas_Overview.ipynb:1]: <p>yeah, we should do that on other nbs too</p>
